.env.example

```diff
@@ -1,2 +1,4 @@
 NEXT_PUBLIC_DISCORD_WIDGET_ID=12345
 NEXT_PUBLIC_DISCORD_INVITE_LINK=https://
+DATABASE_URL=mongodb+srv://username:password@url
+API_KEY=!verySuperSecretKey!
```

Comment:
This and the `withAuthorization` leave me a touch unsure how to work with. We have an API key here, and then for a Bearer authorization token you set it to `Bearer:` followed by a base64 encoded version of the authorization?